src/components/calcite-block/calcite-block.scss

```diff
@@ -17,8 +17,9 @@
 }
 
 .header-container {
-  position: relative;
-  display: flex;
+  display: grid;
+  grid-template: auto / auto auto 1fr;
```

Comment:
This won't work in IE11, but I think that's ok :)

Reply:
There's partial support for it, but I haven't tested at all on IE11.